### ISSUE TITLE
ark sa always needs namespace create perms

### ISSUE THREAD

**Issue by MansM** [heptio/ark#673]
**What steps did you take and what happened:**
We want to run ark without any restore permissions at this moment in time.
We are running openshift 3.9
I created a copy of cluster-reader clusterrole and added:
```
- apiGroups:
  - ark.heptio.com
  resources:
  - '*'
  verbs:
  - '*'
```
when starting ark:
An error occurred: error creating namespace config-hist: namespaces is forbidden: User "system:serviceaccount:config-hist:ark" cannot create namespaces at the cluster scope: User "system:serviceaccount:config-hist:ark" cannot create namespaces at the cluster scope

**What did you expect to happen:**
running ark, able to make backups

**Anything else you would like to add:**

**Environment:**

- Ark version (use `ark version`):
- Kubernetes version (use `kubectl version`):
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`):

**Comment by ncdc** [comment]
@MansM Ark currently attempts to create the namespace in which it runs. We should probably remove this logic and require you to create the namespace first. @skriss @nrb wdyt?

**Comment by MansM** [comment]
Or make the auto creation configurable. Though I wonder how it can even create the namespace. As an deployment should run in a namespace to even be able to access the sa to connect to the api. I can be mistaken....

**Comment by nrb** [comment]
I think requiring the namespace be created beforehand is fine.

**Comment by skriss** [comment]
@ncdc @nrb agree, I think we can remove the attempt to create.  I'll get a patch in for this, and we can include it in v0.9.1 which I think we'll probably try to release next week. Thanks for reporting this @MansM !
